src/Error/Debugger.php

```diff
@@ -522,15 +523,15 @@ protected static function _export($var, int $depth, int $indent): string
 
                 return "'" . $var . "'";
             case 'array':
-                return static::_array($var, $depth - 1, $indent + 1);
+                return static::_array($var, $context->withAddedDepthAndIndent());
```

Comment:
This function name `withAddedDepthAndIndent()` feels like you're increasing max depth, not jumping the context to the next depth level. Not sure there's a much better name for it though.

Reply:
Maybe `atNextDepthAndIndent()` or `withNextDepthAndIndent()`. Just a thought.

Reply:
I couldn't come up with a better name. I chose to use the `with` style naming as the DumpContext is a mostly immutable object. I first implemented it as mutable one and tracking depth/indent got really messy.